tools/validate_configs/golden_copies/expectations/igc_pkr/instructions.txt

```diff
@@ -0,0 +1,32 @@
+# Advanced Deployment Instructions
```

Comment:
If the file is written in md, should it not be called instructions.md?

Reply:
My vote is for `txt`, as current file doesn't contain any Markdown syntax (except `#`), while `txt` extension may appear more inviting for some users to open over `md`.
To make it more `txt`-like we can use alternative Markdown headers syntax:
```markdown
Advanced Deployment Instructions
==========================
```
Advanced Deployment Instructions
==========================

Reply:
Then I vote to leave it as is. :)

Reply:
I did deliberately choose _not_ to use Markdown syntax except where it remained directly legible for command-line users without distractions (esp. backticks).

I do think I prefer the `====` / `----` approach for header levels 1 and 2 and will switch.